multicluster/cmd/multicluster-controller/options.go

```diff
@@ -41,12 +45,31 @@ func (o *Options) complete(args []string) error {
 	var err error
 	options := ctrl.Options{Scheme: scheme}
 	ctrlConfig := &mcsv1alpha1.MultiClusterConfig{}
+	klog.Info("Config file is ", o.configFile)
```

Comment:
i am assuming all these debug logs will be removed .. maybe we can add a TODO so as not to forget .. for after demo